.storybook/main.ts

```diff
@@ -6,7 +6,8 @@ const config: StorybookConfig = {
         '@storybook/addon-links',
         '@storybook/addon-essentials',
         '@storybook/addon-interactions',
-        '@storybook/addon-viewport'
+        '@storybook/addon-viewport',
+        'storybook-dark-mode'
```

Comment:
It seems like storybook has their own addon for witching themes ([@storybook/addon-themes](https://storybook.js.org/addons/@storybook/addon-themes)), did you try it before deciding to use this other one? Checking it's docs they are quite lacking, is that what you meant in the description when saying that storybook doesn't explain theming well? 🤣

Reply:
It only allow changing the theme for preview. We are changing the whole documentation theme together with the preview